src/index.ts

```diff
@@ -230,11 +230,16 @@ function subscribeP<T>(
           if (deepEqual(toJSON(parent.y), parent.p)) {
             return;
           }
-
-          const arrayOps = parseProxyOps(ops);
+          if (!arrayOpsAll) {
+            arrayOpsAll = parseProxyOps(ops);
+          }
+          const pathkey = path.join('.');
+          if (!(pathkey in arrayOpsAll)) return;
+          const arrayOps = arrayOpsAll[pathkey];
```

Comment:
Is the use of `pathkey` important of the refactoring? Maybe so.
I'm not a big fan of usign `.` delimiter. What if a path includes the dot?

Reply:
I don't think it would be of impact here, since we never break on `.` again, it's merely just a concatenated representation for the parent path, so we can use it as a key to refer to a collection of arrayOps together.

Reply:
Technically, `["a.b", "c"]` and `["a", "b.c"]` can conflict.